README.md

```diff
@@ -445,10 +445,20 @@ We have (more than) enough themes for the time being. Please add your theme to t
 
 ## Contributors
 
+<a href="https://github.com/ohmyzsh/ohmyzsh/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ohmyzsh/ohmyzsh" width="100%"/>
+</a>
+
 Oh My Zsh has a vibrant community of happy users and delightful contributors. Without all the time and help from our contributors, it wouldn't be so awesome.
 
 Thank you so much!
 
+## Star History
+
+<a href="https://star-history.com/#ohmyzsh/ohmyzsh">
+  <img src="https://api.star-history.com/svg?repos=ohmyzsh/ohmyzsh&type=Date" alt="Star History Chart" width="100%" />
+</a>
+
```

Comment:
I don't think star history fits us... It can only display the first 40k starts, and then it's a straight line. I'd remove this